src/status_im/ui/screens/wallet/refresh_control.cljs

```diff
@@ -4,12 +4,10 @@
             [quo.react-native :as rn]))
 
 (defn refresh-action []
-  (fn []
-    (when (false? @(re-frame/subscribe [:prices-loading?]))
-      (re-frame/dispatch [:wallet.ui/pull-to-refresh]))))
+  (re-frame/dispatch-sync [:wallet.ui/pull-to-refresh])
+  (reagent/flush))
```

Comment:
@Ferossgp 

I've tried adding/using `flush` everywhere, but not within this function.

I take it this resolves the issue when pulling to refresh that a slight "jump" occurs?
(I wouldn't ask, but getting the immutable nix build without a failure is not happening...)

Reply:
Sorry that it took so long, I was on vacation. Yes, this resolves the jump, as it forces the synchronizing state to react immediately after setting the new value, without waiting for the next animation frame.

Reply:
No problem about the time lag. I was told you were on holiday...

Yea, I figured what it does, but what I found annoying is that I put the `flush` in various where I called the pull to refresh and it didn't prevent the jump. What was your reasoning for adding it in this function? Was it trial & error? Thanks

Reply:
I've put it there because it is needed to call flush immediately after setting the value into DB. Very strange that it hasn't worked, maybe some other executions were in between thus postponing the call of flush

Reply:
Possibly. Thanks for the explanation